packages/x-data-grid-pro/src/tests/detailPanel.DataGridPro.test.tsx

```diff
@@ -389,6 +389,7 @@ describe('<DataGridPro /> - Detail panel', () => {
     const getDetailPanelHeight = spy(() => 100);
     const { setProps } = render(
       <TestCase
+        autoHeight
```

Comment:
this should also be removed, no?

Reply:
Some of the tests failed. I tried to remove as many as I could.